Signal/src/ViewControllers/ConversationView/Cells/OWSAudioMessageView.m

```diff
@@ -193,34 +193,46 @@ - (BOOL)isVoiceMessage
     return (self.attachmentStream.isVoiceMessage || self.attachmentStream.sourceFilename.length < 1);
 }
 
-- (void)createContentsForSize:(CGSize)viewSize
```

Comment:
The audio and generic message views weren't using iOS auto layout, so they didn't properly re-layout when the cell changed size.  That's been fixed in this PR.